src/modules/UI/scenes/Request/Request.ui.js

```diff
@@ -67,6 +74,16 @@ export default class Request extends Component {
     .catch((e) => console.log(e))
   }
 
+  componentWillMount () {
+    this.keyboardWillShowListener = Keyboard.addListener('keyboardWillShow', this.keyboardWillShow.bind(this))
```

Comment:
Android does not have `keyboardWillShow` or `keyboardWillHide`

Reply:
I know, no android animation first